src/hotspot/share/gc/shared/space.cpp

```diff
@@ -99,18 +99,9 @@ void ContiguousSpace::mangle_unused_area_complete() {
 }
 #endif  // NOT_PRODUCT
 
-
-void ContiguousSpace::print_short() const { print_short_on(tty); }
-
-void ContiguousSpace::print_short_on(outputStream* st) const {
-  st->print(" space " SIZE_FORMAT "K, %3d%% used", capacity() / K,
-              (int) ((double) used() * 100 / capacity()));
-}
-
-void ContiguousSpace::print() const { print_on(tty); }
```

Comment:
It is a common pattern in HotSpot to have a `print()` function that delegates to `print_on(tty)`. I think this function should be kept.